README.md

```
@@ -211,6 +211,38 @@ dotnet ef database update --project PatchPanda.Web --startup-project PatchPanda.
+```
+
+PatchPanda sets these variables for your script:
+- `PP_PROJECT_DIR`: Directory of the compose file
+- `PP_NAME`: Container name to be updated
+- `PP_OLD_VERSION`: Previous image version
+- `PP_NEW_VERSION`: New image version
+- `PP_UPDATE_TIME`: ISO timestamp of the update
+
+Keep in mind the update script runs within the PatchPanda container and not on the host itself. However, you have access to the host file system via volumes
+
+## Example post-update hook script
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)